src/Geocoder/Provider/What3wordsProvider.php

```diff
@@ -56,7 +56,8 @@ protected function executeQuery($query)
     {
         $content = $this->getAdapter()->get($query)->getBody();
 
-        if (null === $data = json_decode($content, true)) {
+        $data = json_decode($content, true);
+        if ($data === null || empty($data)) {
```

Comment:
@vicchi It seems that I was wrong in both my comments on #42 

Would you mind giving this PR a review and merge it if you like it